### PR TITLE
WIP: feat(gpu): add GPU Instance doc MTA-1497

### PR DESCRIPTION
**Pull request by RoRoJ**

### CODE REVIEW

File: compute/gpu/how-to/create-manage-gpu-instance.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - step: TODO
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/compute/gpu/create-manage-gpu-instance
+image: /assets/images/docs/scaleway.png
+dates:
+  validation: 2021-10-13
+  posted: 2021-10-13
+---
+
+Scaleway GPU Instances are virtual compute instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, articifical intelligence, rendering and video encoding. After you have created your GPU Instance, you can connect to it via SSH and use one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a preinstalled [Pipenv](/compute/gpu/how-to/use-gpu-with-docker) environment with all your favourite AI libraries and tools pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you have finished with your GPU Instance, you can easily delete it from the Scaleway Console.
```
Comment (nedaehm) [inline compute/gpu/how-to/create-manage-gpu-instance.mdx:14]:
```suggestion
Scaleway GPU Instances are virtual compute Instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, articifical intelligence, rendering and video encoding. After you have created your GPU Instance, you can connect to it via SSH and use one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a preinstalled [Pipenv](/compute/gpu/how-to/use-gpu-with-docker) environment with all your favourite AI libraries and tools pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you have finished with your GPU Instance, you can easily delete it from the Scaleway Console.
```

File: compute/gpu/how-to/use-gpu-with-docker.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com) 
+- You have created a [GPU Instance](https://www.scaleway.com/en/gpu-instances/)
+- You have [generated your SSH key](https://www.scaleway.com/en/docs/console/my-project/how-to/create-ssh-key/)
+
+</Message>
+
+## How to get started on a GPU Instance with Docker
+
+1. [Connect](/compute/instances/how-to/connect-to-instance/) to your instance via SSH
```
Comment (nedaehm) [inline compute/gpu/how-to/use-gpu-with-docker.mdx:26]:
```suggestion
1. [Connect](/compute/instances/how-to/connect-to-instance/) to your Instance via SSH
```

File: compute/gpu/how-to/use-gpu-with-docker.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+</Message>
+
+## How to map volumes
+
+It is recommended to map volumes into your Docker containers. You can either map local volumes on your instance or add block storage volumes to your instance for increased data redundancy.
+
+### How to map local volumes
+
+You can map local storage of your GPU instance into your container using the `-v <local_storage>:<container_mountpoint>` flag. Following an example how to map the directory `/root/mydata` into the directory `/workspace` in the container:
```
Comment (nedaehm) [inline compute/gpu/how-to/use-gpu-with-docker.mdx:45]:
```suggestion
You can map local storage of your GPU Instance into your container using the `-v <local_storage>:<container_mountpoint>` flag. Following an example how to map the directory `/root/mydata` into the directory `/workspace` in the container: 
```

File: compute/gpu/how-to/use-gpu-with-docker.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+```
+
+### How to map Block Storage volumes 
+
+It is also possible to map [Block Storage](https://www.scaleway.com/en/block-storage/) volumes into your containers. Block Storage is fully backed by SSDs for both classes. These three-time replacted, high-speed drives allow up to 5,000 IOPS. 
+Once [attached](/storage/block/how-to/attach-a-volume/) and [mounted](/storage/block/how-to/mount-and-use-volume/) in the host OS of the GPU Instance, you can map the volume like a local volume. 
+
+<Message type="tip">
+
+Block Storage volumes are independent from your GPU instance and provide three-time replicated storage. It is recommended to use Block Storage for storing your datasets, training logs, model source code, etc...
```
Comment (nedaehm) [inline compute/gpu/how-to/use-gpu-with-docker.mdx:58]:
```suggestion
Block Storage volumes are independent from your GPU Instance and provide three-time replicated storage. It is recommended to use Block Storage for storing your datasets, training logs, model source code, etc...
```

File: compute/gpu/how-to/use-gpu-with-docker.mdx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+|---------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-p 8080:80`                    | This flag maps TCP port 80 in the container to port 8080 on the Docker host.                                                                               |
+| `-p 192.168.1.100:8080:80`      | This flag maps TCP port 80 in the container to port 8080 on the Docker host for connections to host IP 192.168.1.100.                                      |
+| `-p 8080:80/udp`                | This flag maps UDP port 80 in the container to port 8080 on the Docker host.                                                                               |
+| `-p 8080:80/tcp -p 8080:80/udp` | This flag maps TCP port 80 in the container to TCP port 8080 on the Docker host, and map UDP port 80 in the container to UDP port 8080 on the Docker host. |
+
+
+## How to access the GPU from the inside of a Docker container
+
+You can access the GPU of your instance from the inside of a Docker container using the `nvidia-docker` tool.
```
Comment (nedaehm) [inline compute/gpu/how-to/use-gpu-with-docker.mdx:100]:
```suggestion
You can access the GPU of your Instance from the inside of a Docker container using the `nvidia-docker` tool. 
```

File: compute/gpu/how-to/use-gpu-with-docker.mdx [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+## How to push and list Docker Images stored on a Scaleway Registry
+
+<Message type="tip">
+
+GPU Instances are compatible with [Scaleway Container Registry](https://www.scaleway.com/en/docs/compute/container-registry/quickstart/). You can use this registry to store and pull your own docker Images.
+
+</Message>
+
+See [detailed information](https://www.scaleway.com/en/docs/compute/container-registry/how-to/pull-images/) on how to pull custom Docker Images on your GPU instance.
```
Comment (nedaehm) [inline compute/gpu/how-to/use-gpu-with-docker.mdx:154]:
```suggestion
See [detailed information](https://www.scaleway.com/en/docs/compute/container-registry/how-to/pull-images/) on how to pull custom Docker Images on your GPU Instance.
```

File: compute/gpu/how-to/use-pipenv.mdx [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+
+  `pipenv update <package>`
+
+  <Message type="tip">
+
+    To update all packages, simply run `pipenv update`
+  
+  </Message>
+
+  TODO CHECK: if the user makes changes to the preinstalled virtual environment, will these persist once the container exits? Would the ai file have to be mapped to a local volume of the instance?
```
Comment (nedaehm) [inline compute/gpu/how-to/use-pipenv.mdx:126]:
```suggestion
  TODO CHECK: if the user makes changes to the preinstalled virtual environment, will these persist once the container exits? Would the ai file have to be mapped to a local volume of the Instance?
```

File: compute/gpu/quickstart.mdx [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  - step: TODO
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/compute/gpu/quickstart
+image: /assets/images/docs/scaleway.png
+dates:
+  validation: 2021-10-13
+  posted: 2021-10-13
+---
+
+Scaleway GPU Instances are virtual compute instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, articifical intelligence, rendering and video encoding. After you have created your GPU Instance, you can connect to it via SSH and use one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a preinstalled [Pipenv](/compute/gpu/how-to/use-gpu-with-docker) environment with all your favourite AI libraries and tools pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you've finished with your GPU Instance, you can easily delete it from the Scaleway Console.
```
Comment (nedaehm) [inline compute/gpu/quickstart.mdx:14]:
```suggestion
Scaleway GPU Instances are virtual compute Instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, articifical intelligence, rendering and video encoding. After you have created your GPU Instance, you can connect to it via SSH and use one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a preinstalled [Pipenv](/compute/gpu/how-to/use-gpu-with-docker) environment with all your favourite AI libraries and tools pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you've finished with your GPU Instance, you can easily delete it from the Scaleway Console.
```

**Comment by bene2k1** [comment]
Content archived on GitLab for future use.

**Pull request closed without merging**